server/registry/internal/storage/gorm/client.go

```diff
@@ -88,15 +86,18 @@ func NewClient(ctx context.Context, driver, dsn string) (*Client, error) {
 		// postgres runs in a separate process and seems to have no problems
 		// with concurrent access and modifications.
 		disableMutex = true
-		c := &Client{db: db}
-		c.ensure()
-		return c, nil
+		return &Client{db: db}, nil
 	default:
 		unlock()
 		return nil, fmt.Errorf("unsupported database %s", driver)
 	}
 }
 
+// EnsureTables ensures that all necessary tables exist in the database.
+func (db *Client) EnsureTables() error {
+	return db.ensureTables()
+}
```

Comment:
Why not export `ensureTables` directly instead of wrapping it?

Reply:
No reason, fixed.